src/Evolution/Initialization/Evolution.hpp

```diff
@@ -70,11 +70,12 @@ struct Evolution {
                                       LocalSystem::is_in_flux_conservative_form>
   struct ComputeTags {
     using type = db::AddComputeTags<
-        ::Tags::Time, ::Tags::DerivCompute<variables_tag,
-                                           ::Tags::InverseJacobian<
-                                               ::Tags::ElementMap<dim>,
-                                               ::Tags::LogicalCoordinates<dim>>,
-                                           typename System::gradients_tags>>;
+        ::Tags::Time,
+        ::Tags::DerivCompute<
+            variables_tag,
+            ::Tags::InverseJacobian<::Tags::ElementMap<dim>,
+                                    ::Tags::Coordinates<dim, Frame::Logical>>,
```

Comment:
FYI, this change is needed because I switched the Jacobian compute item to the `Tags::Coordinates<Dim, Frame::Logical>` base tag.